maven/impl-maven-embedded/src/main/java/org/jboss/shrinkwrap/resolver/impl/maven/embedded/pom/equipped/ConfigurationStageImpl.java

```diff
@@ -242,4 +243,16 @@ public ConfigurationDistributionStage setResumeFrom(String resumeFrom) {
         getInvocationRequest().setResumeFrom(resumeFrom);
         return this;
     }
+
+    @Override
+    public ConfigurationDistributionStage setAlternatePomFile(String pomFile) {
```

Comment:
the content of the method seems to be similar to this one: https://github.com/shrinkwrap/resolver/blob/master/maven/impl-maven-embedded/src/main/java/org/jboss/shrinkwrap/resolver/impl/maven/embedded/pom/equipped/PomEquippedEmbeddedMavenImpl.java#L42-L51
wouldn't be better moving this verification to the `build()` phase? So the pom file is verified before the build execution... Taking into account also the possibility of using `setWorkingDirectory`

Reply:
> Taking into account also the possibility of using setWorkingDirectory

I don't really understand this and have two concerns here at this point.
1. Should we be setting the alternate pom file relative to the working directory, or like you mentioned earlier(as is the case right now), that it is the user's responsibility to specify the complete path ? 
2. `setWorkingDirectory` and `setAlternatePomFile` are more or less of the same nature, except for the fact the `setAlternatePomFile` enables resetting the pom file, whereas `setWorkingDirectory` only has the effect to changing the directory, would you recommend moving the the logic of resetting the pom file to be merged with the `setWorkingDirectory`  ?

Reply:
1. as I have mentioned - this is the user's responsibility
2. thinking about it, abbility to specify a new pom file make sense. Although, I'm not sure if that will be used any time when we have the `setWorkingDirectory` working properly. Keep it as it is

Reply:
my comment was related to the code - the content of this method is very similar to the referenced one. It would be great if we could merge them in the sense of setting final `pom.xml` file before the actual build execution.